### PR TITLE
update checklist

### PR DESCRIPTION
**Pull request by 0xarshia**
I just added some important checks to grow the list level :)

### CODE REVIEW

**Review by 0xdanial — CHANGES_REQUESTED** [review]
so helpful

File: audit-readiness-checklist.md [hunk 1]
```diff
@@ -14,6 +14,19 @@
...
+- [ ]  Only using modifier if necessary in more than one place.
+- [ ]  All types are being explicitly set (e.g. using uint256 instead of uint).
+- [ ]  Using keccak256 instead of the alias sha3.
+- [ ]  Does not use tx.origin anywhere.
+- [ ]  Use revert instead of throw.
+- [ ]  Avoid using problematic features - If you must, be aware of their many nuances
+  - [ ] send (nuances)
+  - [ ] low level functions (call, delegatecall, callcode, inline assembly)
+  - [ ] var
+  - [ ] avoid arbitrary jumps (ref.)
```
Comment (brockelmore) [inline audit-readiness-checklist.md:28]: These are more "code review" style recommendations, that don't follow the general theme of this checklist. Basically all of the above will be highlighted in `slither`. 

Also, I generally don't believe in `address(0)` checks (sometimes fine though).

File: audit-readiness-checklist.md [hunk 2]
```diff
@@ -22,3 +35,5 @@
 - [ ]  You can also try to use formal verification tools to verify the invariants, but be aware that -- in practice -- current formal verification tools are rarely useful for anything non-trivial.
 - [ ]  Write down your security assumptions. This doesn't have to be super formal. E.g., "We assume that the `owner` is not malicious, that the Chainlink oracles won't lie about the token price, that the Chainlink oracles will always report the price at least once every 24 hours, that all tokens that the `owner` approves are ERC20-compliant tokens with no transfer hooks, and that there will never be a chain reorg of more than 30 blocks". This helps you understand how things could possibly go wrong *even if your contracts are bug-free*. Auditors may also be able to tell you whether or not your assumptions are realistic. They may also be able point out assumptions you're making that you didn't realize you were making.
 - [ ]  If you're unsure about something in your own code, or there are areas where you'd like auditors to spend more time, make a list of these to share with the auditors.
+- [ ]  All methods and loops are within the maximum allowed gas limt.
```
Comment (brockelmore) [inline audit-readiness-checklist.md:38]: I believe slither catches this as well

**Review by brockelmore — CHANGES_REQUESTED** [review]
Thanks for the PR!

These additions don't really fit within the theme of the checklist and are something that a team may add to the PR review process and are more inline with a "Common Mistakes" list. Most, if not all, will be caught by `slither`. These also would be caught by an auditor in all likelihood and don't have concrete benefits to the auditor's ability to do their job (which is the main theme of this checklist)

File: audit-readiness-checklist.md (hunk 1 above)
Comment (0xarshia) [reply audit-readiness-checklist.md:28]: thanks for your feedback, okay fine.

**Pull request closed without merging**